pydra/engine/helpers_file.py

```diff
@@ -120,6 +120,7 @@ def template_update(inputs, output_dir, state_ind=None, map_copyfiles=None):
         field
         for field in attr_fields(inputs)
         if field.metadata.get("output_file_template")
+        and getattr(inputs, field.name) is not False
```

Comment:
Do you want it to be specifically `False`, or is falsey better? Consider `Optional[Path]`, where the default is `None` instead. It's unclear when a value would be falsey and useful.

```suggestion
        and getattr(inputs, field.name)
```

I'm not 100% sure what would be least surprising here.

Reply:
The only falsey value would be the empty string, given the field is typed ty.Union[str, bool]. However, in the proposed syntax changes they would change to ty.Union[Path, bool] so the only falsey value would be False in any case

Reply:
Ah, just reread your comment. For Optional[Path] fields, we would want the None values to be omitted wouldn't we? So falsey would be better

Reply:
Double ah, I remembered why it is explicitly not False instead of falsey, we want to use the template when the value is attrs.NOTHING. Have reverted the suggested change